src/components/FormValidator.js

```diff
@@ -78,3 +78,6 @@ export default class FormValidator {
     this.toggleButtonState();
   }
 }
```

Comment:
Please, delete `this.disableButton();` from line 74 in `FormValidator` because you'll not be able to click the button if there is a server error. You should disable only in block `then` after a **successful** response from the server